apps/labs/posts/sangho-internship-blogpost.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Sangho's Internship at Quansight with PyTorch-Ignite project"
+published: October 6, 2022
+author: sangho-lee
+description: 'Blogpost of working on the PyTorch-Ignite project during internship at Quansight'
+category: [Machine Learning, OSS Experience]
+featuredImage:
+  src: /posts/sangho-internship-blogpost/sangho1.png
+  alt: 'An image of the intern, Sangho.'
+hero:
+  imageSrc: /posts/sangho-internship-blogpost/ignite_logo_mixed.png
+  imageAlt: 'PyTorch-Ignite Logo'
+---
+
+Hey, I'm [Sangho Lee], a master's student from Seoul National University.
+I have participated to the [PyTorch-Ignite](https://pytorch-ignite.ai/) project especially with distributed computations at Quansight Labs.
+First thing I contributed was improvements of the test code for metric computation in Distributed Data Parallel (DDP) configuration.
+Second is modifying `all_reduce` and `all_gather` methods to take `group` argument.
+
+## [About PyTorch-Ignite and distributed computations](https://pytorch-ignite.ai/tutorials/advanced/01-collective-communication/)
+
+When you have 1) a larger batch of dataset such as training images or you 2) need to parallelize computations, then it is natural to use multi-gpu in distributed configuration.
```

Comment:
@puhuk Can you rewrite this section using https://pytorch.org/tutorials/beginner/dist_overview.html as source info and maybe others from internet.